bedrock2/Makefile

```diff
@@ -62,7 +64,14 @@ special/BytedumpTest.out: special/BytedumpTest.v special/BytedumpTest.golden.bin
 	diff -u special/BytedumpTest.golden.bin special/BytedumpTest.out.tmp
 	mv special/BytedumpTest.out.tmp special/BytedumpTest.out
 
-force:
```

Comment:
Did you delete this line on purpose?

Reply:
No. What does it do?

Reply:
A target without prerequisites and without recipe is considered always outdated, and by making `Makefile.coq.ex` and `Makefile.coq.noex` depend on it, we can force these to be regenerated each time, so that when we add or delete `.v` files, they automatically get added to/removed from the Makefile. I guess since `force` is phony, generating `Makefile.coq.ex` and `Makefile.coq.noex` still worked on this PR even though no `force` target exists any more, but could you check if `coq_makefile` is also still re-run on each `make` invocation?